app/routes.py

```diff
@@ -1 +1,217 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, request, make_response, abort
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
```

Comment:
You can create a routes directory and create separate route files for task and goal to further organize your project. If you have separate route files then each blueprint could just be called `bp` since you wouldn't need to specific the model (since it's in it's own file)